web_src/less/themes/theme-arc-green.less

```diff
@@ -614,10 +620,11 @@ footer {
 }
 
 .ui.label,
-.ui.label.basic {
+.ui.label.basic,
+.theme-arc-green .ui.label.basic {
```

Comment:
I think this selector may be unnecessary. You generally always want to match the selector used in the light theme and this file is only included if theme is set to arc-green.

Reply:
You're right it's not necessary, I was trying to avoid the precending rule from Fomantic UI, but all of those `.ui.ui.ui` make it more specific and override even having the `.theme-arc-green` selector:

![Screenshot from 2020-10-22 11-07-04](https://user-images.githubusercontent.com/27286/96912198-d0566d00-1456-11eb-9733-e1aafcf9e566.png)

Result:
![image](https://user-images.githubusercontent.com/27286/96912671-8f128d00-1457-11eb-90b9-f3be328fc781.png)

Which forces the white background, and it makes it hard to read bright colors.

Would a good be to specify each of the possible background colors for the Fomantic UI base colors?
![Screenshot from 2020-10-22 11-11-45](https://user-images.githubusercontent.com/27286/96912561-65596600-1457-11eb-8dac-1c764e29be1e.png) (reference: https://fomantic-ui.com/elements/label.html).

Or would that be out of the scope of the theme, as seems like those colors are not being used at all?

Reply:
That white backgrounds are bugs and should be overridden with the exact same selector fomantic uses, even if it has all those `.ui`. Feel free to do that in another PR.